db/schema.rb

```diff
@@ -87,48 +87,32 @@
 
   create_table "dimensions_items", force: :cascade do |t|
     t.string "content_id"
-    t.string "title"
-    t.string "link"
-    t.string "description"
-    t.string "organisation_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "latest"
-    t.index ["content_id", "link", "organisation_id"], name: "dimensions_items_natural_key"
+    t.string "organisation_id"
```

Comment:
It looks you need to regenerate the schema. There should be no `organisation_id` after the migrations have run. `title` should remain.